app_server/app/adapters/socket/SocketAdapter.js

```diff
@@ -100,6 +100,10 @@ Class.closeRoom = function(roomName) {
   this.roomsManager.removeRoom(roomName);
 };
 
+Class.getRooms = function() {
+  return this.roomsManager.getRooms();
+};
```

Comment:
Can't use arrow function here since they don't capture `this` as they are lexical scoped.